src/ASM/ASMs2Dmx.C

```diff
@@ -96,7 +96,7 @@ bool ASMs2Dmx::readBasis (std::istream& is, size_t basis)
 bool ASMs2Dmx::write (std::ostream& os, int basis) const
 {
   if (basis == -1)
-    os <<"200 1 0 0\n" << *proj;
+    os <<"200 1 0 0\n" << *static_cast<const Go::SplineSurface*>(projB);
```

Comment:
No need to cast here, since the `operator<<`  is defined in the base class `Go::Streamable`?